uaclient/tests/test_security_status.py

```diff
@@ -154,18 +163,30 @@ def test_finds_updates_for_installed_packages(
                     "version": "2.0",
                     "service_name": "esm-infra",
                     "status": "pending_attach",
-                }
+                },
+                {
+                    "package": "more_than_one_update_available",
+                    "version": "2.0",
+                    "service_name": "standard-security",
+                    "status": "upgrade_available",
+                },
+                {
+                    "package": "more_than_one_update_available",
+                    "version": "3.0",
+                    "service_name": "esm-infra",
+                    "status": "pending_attach",
+                },
             ],
             "summary": {
                 "ua": {
                     "attached": False,
                     "enabled_services": [],
                     "entitled_services": [],
                 },
-                "num_installed_packages": 3,
-                "num_esm_infra_updates": 1,
+                "num_installed_packages": 5,
```

Comment:
Shouldn't this be `4`?

Reply:
Well no, I have 5 packages with an "installed" version in the test (from last revision, I added a package with a couple updates and another one representing a non-security update).

Reply:
Ah you're right! Looks good then